pkg/api/handler/merkle_verifier_adapter.go

```diff
@@ -0,0 +1,30 @@
+package handler
+
+import (
+	"context"
+
+	"github.com/bitcoin-sv/arc/internal/beef"
+	"github.com/bitcoin-sv/arc/pkg/blocktx"
+)
+
+type merkleVerifier struct {
+	v blocktx.MerkleRootsVerifier
+}
+
+func (a merkleVerifier) Verify(ctx context.Context, request []beef.MerkleRootVerificationRequest) ([]uint64, error) {
```

Comment:
`a` like `api`? Maybe `m` or `mv` will be better?

Reply:
`a` like adapter